tmt/steps/execute/scripts/tmt-reboot

```diff
@@ -1,12 +1,52 @@
 #!/bin/bash
+PATH=/sbin:/usr/sbin:$PATH
+
+create_nextboot_timer() {
+        if [ -z "$NEXTBOOT_VALID_TIME" ] ; then
+                NEXTBOOT_VALID_TIME=180
```

Comment:
Why is NextBoot valid only for a specific time and reset after that time expires? I know, `rstrnt-reboot` does this too, but we found it to be actually harmful ( https://github.com/restraint-harness/restraint/pull/260 ), so I am afraid you are only repeating the same mistake.
To explain why is it a mistake, I will quote from that PR:
> the shutdown itself may take more than 180s so by the time the reboot finally occurs, the BootNext EFI variable is already reset to its previous value. Thus, the machine will not boot to the desired boot target but will boot from some other target (e.g. the network in Beaker) thus possibly breaking the executed job.

What is supposed to happen if the reboot takes more than 180s? AIUI, in this form, the test machine will break. Was that the intent? If so, why?

If my understanding is not correct, please provide an explanation in the commit message and in a comment, right now there is no explanation why this timer is needed, at all.

Reply:
From what I've understood, I would agree with @pcahyna: Let's not introduce this timeout if it possibly can cause problems when the reboot takes more time. @pfdaly, could you please remove the timeout or explain why it is necessary? @juk & @emmawu, could you please add your thoughts here? Would you also recommend to remove the timeout functionality? Or is it worth/better to keep it here? Thanks for any insight you could provide.

Reply:
@juk AIUI (looking at commit https://github.com/restraint-harness/restraint/commit/11facdcfa06dbe43663107d12d65c001d8dc55fc ) the timeout was implemented for `rstrnt-prepare-reboot` (where it makes sense) and got introduced to `rstrnt-reboot` only indirectly, because of the desire (understandable, but in this case misguided) to reuse `rstrnt-prepare-reboot` as a backend for `rstrnt-reboot` to avoid what was likely perceived as a code duplication. The commit message does not explain the reasoning, but I think this hypothesis makes sense.